{{ cookiecutter.package_name }}/.github/workflows/tox-tests.yml

```diff
@@ -0,0 +1,104 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: Python Tests
+
+on: [push, pull_request]
+
+jobs:
+
+  # Set up matrix to run tox tests across lists of os, python version, and tox environment
+  matrix_tests:
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"  # github actions doesn't yet support this natively so add here
+    strategy:
+      matrix:
+        # Github actions supports ubuntu, windows, and macos virtual environments:
+        # https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+        #
+        # Only run on ubuntu by default, but can add other os's to the test matrix here.
+        # For example -- os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest]
+
+        # Test python 3.7 and 3.8 by default.
+        python-ver: [7, 8]
+
+        # Specify which tox environments to test in this list.
+        tox-env: [cov, alldeps, devdeps, astropylts]
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
If the template also use setuptools-scm (?), this should be a full checkout.

Reply:
i changed this to do `fetch_depth: 0` to match what astropy is doing. not a huge deal for small/new repos. helped a lot for some of my older, cruftier ones...